### PR TITLE
Prevent exception from Whoops when caught exception frame is not related to real file

### PR DESCRIPTION
**Pull request by SilverFire**

### CODE REVIEW

**Comment by staabm** [comment]
I guess this is a fix for https://github.com/filp/whoops/issues/598

would be great to have a unit-test covering the change.

@lcharette can you verify the fix?

**Comment by SilverFire** [comment]
Yes, seems to be a fix for #598 :)

> would be great to have a unit-test covering the change.

Done

File: tests/Whoops/Exception/FrameTest.php [hunk 1]
```diff
@@ -103,6 +103,20 @@ public function testGetFileContents()
...
 
+    /**
+     * @covers Whoops\Exception\Frame::getFileContents
+     * @testWith ["[internal]"]
+     *           ["Unknown"]
+     * @see https://github.com/filp/whoops/pull/599
+     */
+    public function testGetFileContentsWhenFrameIsNotRelatedToSpecificFile($fakeFilename)
+    {
+        $data  = array_merge($this->getFrameData(), ['file' => $fakeFilename]);
```
Comment (staabm) [inline tests/Whoops/Exception/FrameTest.php]: I guess we need php 5.x compat here

```suggestion
        $data  = array_merge($this->getFrameData(), array('file' => $fakeFilename));
```

Comment (lcharette) [reply tests/Whoops/Exception/FrameTest.php]: FYI, since you require php 5.5.9  or above, you should be fine.

> Following [] is supported in PHP >= 5.4:

**Comment by lcharette** [comment]
Looks like it would indeed fix #598. I'll be able to test later today.

**Comment by denis-sokolov** [comment]
Thank you for your contribution!

**Pull request merged**